spec/support/transactions_feature_helper.rb

```diff
@@ -1,18 +1,13 @@
 module TransactionsFeatureHelper
-  def fill_transaction_form(rand_num, new_comment, edit = false)
+  def fill_transaction_form(rand_num, edit = false)
     page.should have_css('#modal-window', visible: true)
-    if edit
-      rand_num += 23
-      new_comment += 'Updated'
-    end
-    fill_data_in_fields(rand_num, new_comment)
+    rand_num += 23 if edit
```

Comment:
@sinpai why `23`?

Reply:
WIll change this magic number to some random one